modules/ROOT/partials/configuration/ai_shortcuts.adoc

```diff
@@ -0,0 +1,91 @@
+
+When set to `+[]+` (an empty array), or `+false+`, the AI Assistant shortcuts toolbar button and menu item do not present in the {productname} instance.
+
+When configured with an instance-specific object array, the AI Assistant shortcuts toolbar button and menu item present, and display the configured shortcuts when activated.
+
+*Type:* `+Array+` of `+Objects+`, or `+Boolean+`
+
+*Default value:* 
+[source, js]
+----
+[
+  { title: 'Summarize', prompt: 'Briefly summarize this.' },
+  { title: 'Explain', prompt: 'Give a short explanation of this.' },
+  { title: 'Improve grammar', prompt: 'Correct grammar.' },
+  { title: 'Improve writing', prompt: 'Enhance this text.' },
+  { title: 'Simplify language', prompt: 'Simplify this.' },
+  { title: 'Make longer', prompt: 'Expand this.' },
+  { title: 'Make shorter', prompt: 'Condense this.' },
+  { title: 'Change tone', subprompts: [
+    { title: 'Descriptive', prompt: 'Make this descriptive.' },
+    { title: 'Formal', prompt: 'Make this formal.' },
+    { title: 'Casual', prompt: 'Make this casual.' },
+    { title: 'Humour', prompt: 'Add humour to this.' },
+    { title: 'Inspirational', prompt: 'Inspire reader with this.' },
+    { title: 'Persuasive', prompt: 'Make this persuasive.' },
+  ] },
+  { title: 'Change style', subprompts: [
+    { title: 'Business', prompt: 'Make this business-like.' },
+    { title: 'Legal', prompt: 'Legal style this.' },
+    { title: 'Journalism', prompt: 'Journalistic style this.' },
+    { title: 'Medical', prompt: 'Medical style this.' },
+    { title: 'Poetic', prompt: 'Make this poetic.' },
+  ] },
+]
+----
+
+NOTE: The default shortcuts are subject to change in an upcoming release. If you intend to keep these shortcuts, include them within your integration.
+
+
+=== Example: using `ai_shortcuts` to present a customised set of AI Assistant shortcuts
+
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea',  // change this value according to your html
+  ai_shortcuts: [
+    { title: 'Classical', subprompts: 
+      [
+      { title: 'Dialogue', prompt: 'Convert this to a Socratic dialogue.' },
+      { title: 'Homeric', prompt: 'Convert this to a Classical Epic.' },
+      ]
+    },
+    { title: 'Celtic', subprompts:
+      [
+      { title: 'Bardic', prompt: 'Convert this to Bardic verse.' },
+      { title: 'Filí', prompt: 'Convert this to Filí-an verse.' },
+      ]
+    },
+  ]
+});
+----
+
+=== Example: disabling `ai_shortcuts`
+
+To disable the AI Assistant shortcuts menu and toolbar options, set `ai_shortcuts` to `false` (or to `+[]+`, an empty array).
+
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea',  // change this value according to your HTML
+  ai_shortcuts: false
+});
+----
+
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea',  // change this value according to your HTML
+  ai_shortcuts: []
+});
+----
```

Comment:
Should we include at min the `plugin` name and `toolbar` config to these examples such as:

```suggestion
[source,js]
----
tinymce.init({
  selector: 'textarea',  // change this value according to your HTML
  plugins: 'ai',
  toolbar: 'aidialog',
  ai_shortcuts: false
});
----

[source,js]
----
tinymce.init({
  selector: 'textarea',  // change this value according to your HTML
  plugins: 'ai',
  toolbar: 'aidialog',
  ai_shortcuts: []
});
----
```
@MitchCTiny can you confirm?

Reply:
@brian-forte - It might also be worth mentioning that by default , when the `ai` plugin is set in the config, but the `ai_shortcuts` has been removed from the `toolbar` config, the `ai_shorcuts` will still present itself in the `menubar` by default. 

After some testing, it looks like  you still need to explicitly remove the `ai_shortcuts` by adding either `ai_shortcuts: []` or `ai_shortcuts: false` to the config after removing `ai_shortcuts` from the toolbar config.

Reply:
This would be unnecessary since both buttons are included in the default toolbar.